src/control/server/server.go

```diff
@@ -53,7 +54,6 @@ func genFiAffFn(fis *hardware.FabricInterfaceSet) config.EngineAffinityFn {
 
 		fi, err := fis.GetInterfaceOnNetDevice(iface, prov)
 		if err != nil {
-			return 0, err
```

Comment:
Do nothing?

Reply:
surprising this doesn't get caught by the server unit tests!